app/views/expenses/new.slim

```diff
@@ -3,6 +3,8 @@
 
   = form.input :amount
 
+  = form.association :category, collection: Category.all
```

Comment:
This is coupling the view with the `Category` model, and also with the `all` query. If you wish to add a filter, for instance, you have to add more code to the view. It is a good rule of thumb to inject all dependencies to the view through instance variables.